airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -208,6 +211,12 @@ def retry_factor(self) -> float:
         """
         return 0 if IS_TESTING else super(StripeStream, self).retry_factor
 
+    def get_cursor(self) -> Optional[Cursor]:
+        parent_cursor = super().get_cursor()
+        if isinstance(parent_cursor, (ResumableFullRefreshCursor, SubstreamResumableFullRefreshCursor)):
+            return None
```

Comment:
it's probably safer to only zero out the RFR cursors, but I don't think for the existing legacy `HttpStream` classes, that we'll ever have any other type of cursor assigned. So it would suffice to just have `return None` by itself. Basically saying that all the legacy `StripeStream` classes have no cursor defined.

Unfortunately this does get a bit muddled up by the concurrent `AbstractStream` having it's own separate `def cursor()` method... But we are working on consolidating the cursor interfaces